curriculum/challenges/english/25-front-end-development/lecture-understanding-performance-in-web-applications/67d2f778a856c37bd31c5022.md

```diff
@@ -12,7 +12,7 @@ Watch the video or read the transcript and answer the questions below.
 
 # --transcript--
 
-What are some key metrics for measuring performace?
+What are some key metrics for measuring performance?
```

Comment:
You haven't found all of the typos. 

There is a typo in the last paragraph.
Please re read through the issue to resolve the problem